cli/tsc/dts/lib.deno_url.d.ts

```diff
@@ -5,7 +5,19 @@
 /// <reference no-default-lib="true" />
 /// <reference lib="esnext" />
 
-/** @category URL */
+/**
+ * Iterator for the URLSearchParams class used to iterate over key-value pairs in search parameters.
```

Comment:
this should say "query string" somewhere - I can imagine someone googling for "deno query string parser" and ideally they'd land here. 

Currently this isn't the case : https://www.google.com/search?q=deno+query+string+parser